src/prefect/cli/worker.py

```diff
@@ -101,6 +101,20 @@ async def start(
         case_sensitive=False,
     ),
 ):
+    """ "
+    Checks if a work-pool is paused or not
+    """
+
+    is_paused = await _check_work_pool_paused(work_pool_name)
+    if is_paused:
+        app.console.print(
+            (
+                f"The work pool {work_pool_name!r} is currently paused. This worker"
+                " will not execute any flow runs until the work pool is unpaused."
+            ),
+            style="yellow",
+        )
+
     """
     Start a worker process to poll a work pool for flow runs.
     """
```

Comment:
The code should start after the docstring.
```suggestion
    """
    Start a worker process to poll a work pool for flow runs.
    """
    is_paused = await _check_work_pool_paused(work_pool_name)
    if is_paused:
        app.console.print(
            (
                f"The work pool {work_pool_name!r} is currently paused. This worker"
                " will not execute any flow runs until the work pool is unpaused."
            ),
            style="yellow",
        )
```